tests/integration/suite/test_pod_security_policies.py

```diff
@@ -43,6 +43,35 @@ def create_pspt(client):
     return pspt
 
 
+def set_psp_enabled(admin_mc, value):
```

Comment:
Rename this function to drop the enabled, and move it to conftest.py so it can be integrated with a fixture for cleanup.